src/FlameGraph.js

```diff
@@ -53,10 +59,17 @@ export default class FlameGraph extends PureComponent<Props, State> {
   };
 
   render() {
-    const { data, height, width } = this.props;
+    const { data, height, onMouseOut, onMouseOver, width } = this.props;
     const { focusedNode } = this.state;
 
-    const itemData = this.getItemData(data, focusedNode, this.focusNode, width);
+    const itemData = this.getItemData(
+      data,
+      focusedNode,
+      this.focusNode,
+      onMouseOut,
+      onMouseOver,
+      width
+    );
```

Comment:
I was unsure of whether this was the correct approach here, or whether I should instead use `outerElementType` and pass it through that, as it's not related to the particular node.